app/controllers/async_request/jobs_controller.rb

```diff
@@ -27,12 +34,17 @@ def render_invalid_token
       render json: { errors: [{ message: 'Invalid token' }] }, status: :bad_request
     end
 
-    def render_pending(job)
-      render json: { status: job.status }, status: :accepted
+    def render_pending(_job)
```

Comment:
Why keeping the param?

Reply:
no need. I forgot to remove that argument